ush/rocoto/workflow_utils.py

```diff
@@ -272,6 +272,9 @@ def source_configs(configs, tasks):
         elif 'wave' in task:
             files.append(find_config(f'config.wave', configs))
             files.append(find_config(f'config.{task}', configs))
+        elif 'gefs' in task:
+            files.append(find_config(f'config.gefs', configs))
+            files.append(find_config(f'config.{task}', configs))
```

Comment:
There are no gefs tasks currently, so this is a placeholder.
Since there is work ongoing on refactoring setup_workflow, this could be held till actual tasks are being added.

Reply:
Currently, I use the existing getic, init and fcst tasks for GEFS, later, I will add recenter task for GEFS. OK, I can delete these lines until actual tasks are being added